addons/udes_stock/models/uom_uom.py

```diff
@@ -0,0 +1,18 @@
+import decimal
+from odoo import api, fields, models
+
+
+class UoM(models.Model):
+    _inherit = "uom.uom"
+
+    decimal_places = fields.Integer(
```

Comment:
By standard field should have u_ at the beginning

Reply:
I think I am going to move this to `udes_uom` too